mod-source-record-storage-server/src/main/java/org/folio/services/handlers/AbstractPostProcessingEventHandler.java

```diff
@@ -73,13 +74,15 @@ public abstract class AbstractPostProcessingEventHandler implements EventHandler
   private final KafkaConfig kafkaConfig;
   private final MappingParametersSnapshotCache mappingParamsCache;
   private final Vertx vertx;
+  private final RecordService recordService;
 
-  public AbstractPostProcessingEventHandler(final RecordDao recordDao, KafkaConfig kafkaConfig,
+  public AbstractPostProcessingEventHandler(final RecordDao recordDao, KafkaConfig kafkaConfig, RecordService recordService,
```

Comment:
@RuslanLavrov  Could you please describe - does the main reason to change using dao on service here - is to avoid retrieving incoming record by instanceId. Am I right?